pkg/virt-handler/vm.go

```diff
@@ -181,6 +181,31 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 
 	oldStatus := vmi.DeepCopy().Status
 
+	if domain != nil {
+		var interfaces = vmi.DeepCopy().Status.Interfaces
```

Comment:
I think the `VirtualMachineInstanceNetworkInterface` is missing a `Name` field. It should look like this:
```yaml
type VirtualMachineInstanceNetworkInterface struct {
        // Name of the interface this status belongs to
        Name string `json:"name"`
	// IP address of a Virtual Machine interface
	IP string `json:"ipAddress,omitempty"`
	// Hardware address of a Virtual Machine interface
	MAC string `json:"mac,omitempty"`
}
```

As a consequence users and our application can reliably detect which interface status belongs to which interface. Since also the interface name matches the qemu device alias on the domain specification, we don't have to guess on the bridge name then.

@mmirecki could you add that field?

Reply:
sure